lib/openstudio-standards/btap/structure.rb

```diff
@@ -303,7 +303,7 @@ def initialize(model = nil, activity = nil, massive = true)
       cat = activity.category
       lload = activity.liveload
 
-      if cat.respond_to?(:to_s)
+      if cat.respond_to?(:to_sym)
```

Comment:
Preferable to check for symbols than _stringable_ objects, then convert to strings.